finality-aleph/src/sync/handler/mod.rs

```diff
@@ -559,9 +569,175 @@ mod tests {
         result
     }
 
+    fn assert_dangling_branch_required(
+        handler: &Handler<
```

Comment:
add 
```
type TestHandler = Handler<...>;
```
pls

Reply:
of course we have `MockHandler`, but I'm blind... fixed, renamed to `TestHandler`